<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipientRegister</class>
 <widget class="QDialog" name="RecipientRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RecipientRegister{background-color: rgb(224, 241, 255);}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>501</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{


    background: #fff;
    margin: 0 auto;
    border-radius: 20px;
	background-color:rgb(70,130,180);

}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label
{
    display: block;
    font-size: 1.5em;
    font-weight: bold;
color:white;

}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitRecipient">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#submitRecipient{
background-color: white;
color:black;
    border-width: 2px;
    border-radius: 10px;

    font: bold 14px;

    padding: 6px;
}

#submitRecipient:hover{
background-color: rgb(224, 241, 255);
}</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameRecipient">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#NameRecipient{

border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="doctorRecipient">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#doctorRecipient{
border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
}</string>
    </property>
    <property name="placeholderText">
     <string>Doctor Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_2{

color:white;
}
</string>
    </property>
    <property name="text">
     <string>Blood Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hospitalRecipient">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#hospitalRecipient{
border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;}</string>
    </property>
    <property name="placeholderText">
     <string>Hospital Name</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AgeRecipient">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_3{
color:white;
}</string>
    </property>
    <property name="text">
     <string>Nearest Hospital</string>
    </property>
   </widget>
   <widget class="QComboBox" name="RecipientBloodType">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB-</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="AgeLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#AgeLabel{
color:white;
}</string>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_4{
color:white;
}</string>
    </property>
    <property name="text">
     <string>Doctor Of the case</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordRecipient">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#PasswordRecipient{
border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QComboBox" name="genderRecipient">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="emailRecipient">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#emailRecipient{

border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
